node/hyperbahn/handler.js

```diff
@@ -24,10 +24,14 @@
 
 var assert = require('assert');
 var util = require('util');
+var fs = require('fs');
+var path = require('path');
 var setTimeout = require('timers').setTimeout;
 
 var Errors = require('../errors.js');
 var TChannelJSON = require('../as/json');
+var TChannelThrift = require('../as/thrift');
```

Comment:
Apache thrift has been used in production for months. 

The discovery feature isn't urgent. No need to rush. Let's just do it right. Thrift is for life, you get one shot at the interface.